integrations/observability/nginx/getting-started/nginx-getting-started-1.0.0.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "nginx",
+  "version": "1.0.0",
+  "displayName": "Nginx-Logs",
+  "description": "Getting Started With Nginx access logs.",
+  "license": "Apache-2.0",
+  "type": "logs",
+  "labels": ["Observability", "Logs", "Fluent-bit"],
+  "author": "OpenSearch",
+  "sourceUrl": "https://github.com/opensearch-project/dashboards-observability/tree/main/server/adaptors/integrations/__data__/repository/nginx/info",
+  "workflow": [
+    {
+      "name": "Fluent-Bit Parser",
+      "label": "Log Parsing",
+      "info": "<info URL>",
+      "description": "Setup Fluent-Bit parser config file parsing Nginx access log fields",
+      "content": "[PARSER]\n    Name   apache\n    Format regex\n    Regex  ^(?<host>[^ ]*) [^ ]* (?<user>[^ ]*) \\[(?<time>[^\\]]*)\\] \"(?<method>\\S+)(?: +(?<path>[^\\\"]*?)(?: +\\S*)?)?\" (?<code>[^ ]*) (?<size>[^ ]*)(?: \"(?<referer>[^\\\"]*)\" \"(?<agent>[^\\\"]*)\")?$\n    Time_Key time\n    Time_Format %d/%b/%Y:%H:%M:%S %z\n\n[PARSER]\n    Name   nginx\n    Format regex\n    Regex ^(?<remote>[^ ]*) (?<host>[^ ]*) (?<user>[^ ]*) \\[(?<time>[^\\]]*)\\] \"(?<method>\\S+)(?: +(?<path>[^\\\"]*?)(?: +\\S*)?)?\" (?<code>[^ ]*) (?<size>[^ ]*)(?: \"(?<referer>[^\\\"]*)\" \"(?<agent>[^\\\"]*)\")\n    Time_Key time\n    Time_Format %d/%b/%Y:%H:%M:%S %z\n"
+    },
+    {
+      "name": "Fluent-Bit Log Converter",
+      "label": "Log Parsing",
+      "info": "<info URL>",
```

Comment:
Are these supposed to still be placeholders?

Reply:
thanks - moved into draft util all the comments / questions are resolved